hosted/Dockerfile

```diff
@@ -1,40 +1,46 @@
-FROM ubuntu:focal
+FROM ubuntu:focal as build
 
 ENV LC_ALL=C.UTF-8
 ENV LANG=C.UTF-8
 ARG HAPPY_ENV
 ENV HAPPY_ENV=$HAPPY_ENV
+ENV STACK_NAME="roaga-somaexplorer"
 ENV DEPLOYMENT_STAGE=$HAPPY_ENV
 
 ENV DEBIAN_FRONTEND=noninteractive
 
-ADD hosted/requirements.txt .
-ADD hosted/create_env_config.py .
-
-ADD server/ server/
-ADD client/ client/
-ADD Makefile Makefile
-ADD common.mk common.mk
-ADD environment.default.json environment.default.json
-
 RUN apt-get update && \
     apt-get install -y gettext moreutils build-essential libxml2-dev python3-dev python3-pip zlib1g-dev python3-requests python3-aiohttp llvm jq npm git zip && \
-    rm -rf /var/lib/apt/lists/*
-
-RUN update-alternatives --install /usr/bin/python python /usr/bin/python3.8 1
-
-RUN python3 -m pip install psycopg2-binary
+    rm -rf /var/lib/apt/lists/* && \
+    apt-get clean && \
+    update-alternatives --install /usr/bin/python python /usr/bin/python3.8 1 && \
+    python3 -m pip install psycopg2-binary 
```

Comment:
I would remove this. `psycopg2` is a driver for Postgres which isn't used by the explorer.